zombies.js

```diff
@@ -79,6 +95,119 @@
  * @property {method} getMaxHealth         Returns private variable `maxHealth`.
  */
 
+class Player {
+  constructor(name, health, strength, speed) {
+    this.name = name;
+    this.health = health;
+    this.strength = strength;
+    this.speed = speed;
+    this.isAlive = true;
+    this.equipped = false;
+    this._pack = [];
+    this._maxHealth = 100;
+  }
+  getPack() {
+    return this._pack;
+  }
+
+  getMaxHealth() {
+    return this._maxHealth;
+  }
+
+  checkPack() {
+    let pack = this.getPack();
+    console.log(pack);
+    // does it need return before console.log?
+    // what's the diff between this.getPack() vs Player.getPack? Should we use one over the other?
```

Comment:
They both refer to the same method, it just depends where your calling from. If you're within the Player itself, you have to use `this`, if you're outside the player instance, then you have to do `ed.getPack()`